sdks/java/container/license_scripts/license_script.sh

```diff
@@ -25,15 +25,23 @@ pip install PyYAML==5.3
 pip install tenacity==5.0.4
```

Comment:
BTW, I wanted to mention that doing a pip install/uninstall like this messes with the user's environment.
The standard way to run this would be via tox (see sdks/python/tox.ini), which will take care of creating a virtualenv to run this under (and you can specify a certain python version if you wish).

Reply:
I reviewed using tox here, but it is a little complicated. We want to keep it simple within a script, so it's easy to use and maintain. Current scripts run at local machine, (not within a docker), so docker will still have clear environment.

Reply:
I don't agree that it's acceptable to modify the user environment. Also, on Jenkins this might cause failure if this script runs concurrently with itself.
I've opened a bug for this: https://issues.apache.org/jira/browse/BEAM-9797

Reply:
Thank you Udi for bringing up this issue. Yes, we should fix it. I will merge this PR and work on a different PR to fix it.